lxd/backup/backup_utils.go

```diff
@@ -28,3 +31,15 @@ func TarReader(r io.ReadSeeker) (*tar.Reader, context.CancelFunc, error) {
 
 	return tr, cancelFunc, nil
 }
+
+// Lifecycle emits a backup-specific lifecycle event.
+func Lifecycle(s *state.State, inst Instance, name string, action string, ctx map[string]interface{}) error {
+	prefix := "instance-backup"
+	u := fmt.Sprintf("/1.0/instances/%s/backups/%s", url.PathEscape(inst.Name()), url.PathEscape(name))
+
+	if inst.Project() != project.Default {
+		u = fmt.Sprintf("%s?project=%s", u, url.PathEscape(inst.Project()))
```

Comment:
@stgraber should be `QueryEscape` as there are slightly different rules for query string params